src/confluence/subcommands/validator.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
```

Comment:
Two things:
1. Please move this file to `core/validator.py`.
2. Replace this file with `validate.py`. It is more declarative and fits the cli `confluence merge...`, `confluence validate ...`, which are instances of the more general `confluence [verb] ...`. Basically copy-and-paste `merge.py` and set up the command line structure for `confluence validate`. Also, add a `--validate` option to `confluence merge` that gives the user the option to validate the output.